### DISCUSSION TITLE
ZF1-Future seems to be abandoned?

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#4255] [General]
@sreichel has created several PRs that solve problems and it would have been nice to receive our support. ZF1-Future seems to be an abandoned project, the last update was in February 2024, PRs and reported issues are no longer answered.

ZF1-Future is vital to OpenMage. Do you consider that we should make a fork to update OpenMage internally, to solve our problems and to collect only certain PRs from their repository?

**Comment by sreichel** [comment]
I dont think it is abandoned. All open PRs have requests for changes (my not atm). For small fixes we can still use patches.

**Reply by addison74** [reply]
In the meantime, since my post, there have been numerous discussions regarding compatibility with PHP versions > 8.1. Let's hope that it will not remain at the discussion stage. 

@sreichel - Your PR's are welcome and I did not hesitate to approve them.

**Comment by sreichel** [comment]
Closed. ZF1Future is still active.
